src/common/libkvs/treeobj.c

```diff
@@ -359,6 +359,23 @@ json_t *treeobj_create_val (const void *data, int len)
     return obj;
 }
 
+json_t *treeobj_create_val_base64 (const char *data)
+{
+    json_t *obj = NULL;
+
+    if (!data)
```

Comment:
should that be an EINVAL?

Should `treeobj_create_val()` be refactored to call this rather than duplicating this code?